source/getting-started/sec-OOP.ptx

```diff
@@ -58,6 +62,6 @@
   </sage>
   
   <p>
-    While this concept might seem abstract at first, it becomes clearer as we dive deeper into Sage. We will learn how Sage utilizes OOP principles and built-in classes, offering a structured way to represent data and perform powerful mathematical operations.
+    While OOP might seem abstract at first, it will become clearer as we dive deeper into Sage. We will see how Sage utilizes OOP principles and built-in classes to offer a structured way to represent data and perform powerful mathematical operations. For a more in-depth look at OOP, check out the <url href="https://doc.sagemath.org/html/en/thematic_tutorials/tutorial-objects-and-classes.html">Sage tutorial</url>
```

Comment:
Not sure about including links. I think that we are not including links in general, this would appear disruptive to me and not consistence with the general style.